packages/engine/src/scene/constants/UVOLTypes.ts

```diff
@@ -208,7 +208,7 @@ export interface GeometryInput {
   targets: Record<string, GLBTarget | DRACOTarget | UniformSolveTarget>
 }
 
-export type TextureFormat = 'ktx2' | 'astc/ktx2'
+export type TextureFormat = 'ktx2' | 'astc/ktx2' | 'vp9'
```

Comment:
Does the fact that the video is encoded in `vp9` actually matter? Do we only support vp9? Couldn't we also support vp8 or h264 or h265 without any other changes to our code?

Reply:
It doesn't actually. Any mp4 video works.

Reply:
Looks like you did that already, thanks!